docs/commands/zq.md

```diff
@@ -95,19 +95,19 @@ Note here that the query `1+1` [implies](../language/dataflow-model.md#implied-o
 |  Option   | Auto | Specification                            |
 |-----------|------|------------------------------------------|
 | `arrows`  |  yes | [Arrow IPC Stream Format](https://arrow.apache.org/docs/format/Columnar.html#ipc-streaming-format) |
-| `json`    |  yes | [JSON RFC 8259](https://www.rfc-editor.org/rfc/rfc8259.html) |
 | `csv`     |  yes | [CSV RFC 4180](https://www.rfc-editor.org/rfc/rfc4180.html) |
+| `json`    |  yes | [JSON RFC 8259](https://www.rfc-editor.org/rfc/rfc8259.html) |
 | `line`    |  no  | One string value per input line |
 | `parquet` |  yes | [Apache Parquet](https://github.com/apache/parquet-format) |
 | `tsv`     |  yes | [TSV - Tab-Separated Values](https://en.wikipedia.org/wiki/Tab-separated_values) |
 | `vng`     |  yes | [VNG - Binary Columnar Format](../formats/vng.md) |
-| `zson`    |  yes | [ZSON - Human-readable Format](../formats/zson.md) |
-| `zng`     |  yes | [ZNG - Binary Row Format](../formats/zson.md) |
-| `zjson`   |  yes | [ZJSON - Zed over JSON](../formats/zjson.md) |
 | `zeek`    |  yes | [Zeek Logs](https://docs.zeek.org/en/master/logs/index.html) |
+| `zjson`   |  yes | [ZJSON - Zed over JSON](../formats/zjson.md) |
+| `zng`     |  yes | [ZNG - Binary Row Format](../formats/zson.md) |
+| `zson`    |  yes | [ZSON - Human-readable Format](../formats/zson.md) |
 
-The input format is typically detected automatically and the formats for which
-`Auto` is `yes` in the table above support _auto-detection_.
+The input format is typically [detected automatically](#auto-detection) and the formats for which
+"Auto" is "yes" in the table above support _auto-detection_.
```

Comment:
This is just a style thing, but I tend to reserve the fixed width font for things a user types, formal language constructs, names of shell tools, etc.